components/clm/src/main/clmfates_interfaceMod.F90

```diff
@@ -211,6 +211,12 @@ module CLMFatesInterfaceMod
    ! developer will at least question its usage (RGK)
    private :: hlm_bounds_to_fates_bounds
 
+   ! The GetAndSetTime function is used to get the current time from the CLM 
+   ! time procedures and then set to the fates global time variables during restart, 
+   ! init_coldstart, and dynamics_driv function calls
```

Comment:
The `init_coldstart` subroutine should have `GetAndSetTime` added to it's begging as well.

Reply:
It should get called between these two lines: https://github.com/E3SM-Project/E3SM/blob/e277f97cef0043babcb4aa87ee01da976c2c7637/components/clm/src/main/clmfates_interfaceMod.F90#L1320-L1322

Reply:
thanks, will add